### PR TITLE
Added release workflow for Android

### LINKED ISSUE #67: Publish app to Play Store
### DoD:
- build release 
- publish app bundle to Twake

### PR DESCRIPTION
**Pull request by tk-nguyen**
Resolves #67

### CODE REVIEW

**Comment by hoangdat** [comment]
- [ ] Conflict with firebase configuration, need to disable firebase first.

**Comment by tk-nguyen** [comment]
Relevent errors:

```bash
e: file:///Users/thainguyen/work-stuff/twake/twake-on-matrix/android/app/src/main/kotlin/chat/fluffy/fluffychat/FcmPushService.kt:3:20 Unresolved reference: fcm_shared_isolate
e: file:///Users/thainguyen/work-stuff/twake/twake-on-matrix/android/app/src/main/kotlin/chat/fluffy/fluffychat/FcmPushService.kt:17:24 Unresolved reference: FcmSharedIsolateService
e: file:///Users/thainguyen/work-stuff/twake/twake-on-matrix/android/app/src/main/kotlin/chat/fluffy/fluffychat/FcmPushService.kt:18:5 'getEngine' overrides nothing
e: file:///Users/thainguyen/work-stuff/twake/twake-on-matrix/android/app/src/main/kotlin/chat/fluffy/fluffychat/FcmPushService.kt:19:30 Unresolved reference: getApplicationContext
```

File: scripts/prepare-android-release.sh [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-echo "keyPassword=${FDROID_KEY_PASS}" >> key.properties
-echo "keyAlias=key" >> key.properties
-echo "storeFile=../key.jks" >> key.properties
-echo $PLAYSTORE_DEPLOY_KEY >> keys.json
-ls | grep key
-bundle install
-bundle update fastlane
-bundle exec fastlane set_build_code_internal
-cd ..
+# git apply ../scripts/enable-android-google-services.patch
```
Comment (hoangdat) [inline scripts/prepare-android-release.sh:2]: IMO, it is disabled with it

Comment (tk-nguyen) [reply scripts/prepare-android-release.sh:2]: This should be resolved after rebasing.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**